docs/roadmap.md

```diff
@@ -0,0 +1,57 @@
+---
+id: roadmap
+title: Open Source Readiness 2022 Roadmap
+sidebar_label: OSR 2022 Roadmap
+---
+
+This document contains an inital Open Source Readiness roadmap as contributed in [2022 Community Roadmap #23](https://github.com/finos/open-source-readiness/discussions/23#discussioncomment-2253788).
+
+## High Level Description
+
+At a high level, OSR should be an engine to identify shared, financial services specific, challenges to open source participation from financial institutions.
+
+The SIG should prioritize the challenges and then foster the creation of projects that address them.
+
+## OSR Responsibilities
+
+The following lists the 2022 responsibilities of the Open Source Readiness SIG.
+
+### Prioritizing common areas of challenge for financial institutions in progressing through the OS maturity model
+
+- Through prioritization of a backlog (e.g. github issues) at OSR SIG meetings.
+- Processing output of the [OSMM](https://github.com/finos/osmm) survey responses FINOS receives.
+- Through real-life findings FINOS can feed in (e.g. Member XX cannot use DCOs, Member YY can’t access github, etc).
+
+### Explore and develop regulators role in the Open Source maturity of the industry, in conjunction with the FINOS Innovation Regulation SIG
+
+- Between [this discussion](https://groups.google.com/a/finos.org/g/osr/c/GMMD38RkaMY/m/nYMwTdR3BAAJ) on the list and this [github discussion](https://github.com/finos/open-source-readiness/discussions/9), consolidating a shared list of common regulatory requirements industry is concerned about
+- Then, mapping to existing solutions (or start open sources project to address them).
+
+### Support the definition and evolutionj of a truly industry-wide OSMM
+
+- Provide / solicit ongoing Feedback and validation of OSMM ahead of FINOS OSMM launch
+- Participate and amply FINOS OSMM launch
+- Promote participants and adjacent communities to take the survey
+- Propose training / certifications to strengthen the value proposition of OSMM for all constituents
+
+### Promote the importance of OSR in FSI and create proven-to-be-successful solutions
+
+- Produce case studies of how FSIs are successfully moving through the OSR journey
+- Source speakers and content for FINOS OSR Marketing (e.g. meetups, blogs, papers, OSSF)
+- Spin off projects to address common areas of challenge as prioritized by the SIG (see point above)
+
+### Define and own relationship with peer (e.g.Reg SIG, Innersource SIG, DevOps SIG) and upstream initiatives (e.g. TODO group, OpenSSF)
+
+# Implementing the OSR SIG Roadmap
+
+To help accelerate the OSR SIG roadmap, the following labels have been added to the OSR repo in GitHub that can be applied to GitHub Issues and Pull Requests.
+
+### Open Source Readiness GitHub Labels
+
+- `osr-relationship-building`
+- `open-source-readiness-promotion`
+- `open-source-maturity-model`
+- `reglatory-requirement`
+- `finacial-institution-challenge`
```

Comment:
Typo, should be 'financial'

Reply:
Fixed here .. 92570273d8307adfa79f0869d264e2af57cf749f